cli/debug.go

```diff
@@ -50,10 +50,10 @@ func writeTerragruntDebugFile(terragruntOptions *options.TerragruntOptions, terr
 	terragruntOptions.Logger.Debugf("Variables passed to terraform are located in \"%s\"", fileName)
 	terragruntOptions.Logger.Debugf("Run this command to replicate how terraform was invoked:")
 	terragruntOptions.Logger.Debugf(
-		"\tterraform %s -var-file=\"%s\" \"%s\"",
+		"\tterraform -chdir=\"%s\" %s -var-file=\"%s\" ",
```

Comment:
Will be helpful to have a test that will track that correct command is generated